### ISSUE TITLE
Azure Storage Emulator (Azurite) calls fail due to legacy `x-ms` headers injection

### ISSUE THREAD

**Issue by yoadsn** [microsoft/ApplicationInsights-node.js#646]
Hi,
Similar to this #358
When working with a local storage emulator the legacy correlation header causes the "Shared Key" authentication mode to fail.
The "domain" to exclude here is the local IP - any chance we can disable those legacy headers easily without excluding the entire local IP address requests?
ATM - I disabled correlation for local runs but this problem was really hard to find... As correctly identified here:
https://github.com/Microsoft/ApplicationInsights-dotnet-server/issues/887

Thanks!

**Comment by markwolff** [comment]
Hi, I'd recommend just disabling these legacy headers via new config option (1.8.1)
```
appInsights.defaultClient.config.ignoreLegacyHeaders = true
```

**Comment by yoadsn** [comment]
Thanks - tested.
For future ref this is available as of v1.8.1 (This is not listed on the README)
